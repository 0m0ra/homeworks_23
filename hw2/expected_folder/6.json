{
    "min age": 26,
    "max age": 26,
    "avg age": 26,
    "median age": 26,
    "< 2 days": 0,
    "< 1 week": 0,
    "< 1 month": 0,
    "< 6 months": 0,
    "> 6 months": 100.0
}