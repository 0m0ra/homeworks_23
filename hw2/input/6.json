{
    "user": {
        "region": "Sochi",
        "registered": "2012-12-24",
        "last_login": "2023-10-01",
        "email": "user@yandex.ru",
        "age": 55
    },
    "user2": {
        "region": "Novosibirsk",
        "registered": "2022-10-30",
        "last_login": "2023-11-21",
        "email": "user2@gmail.com",
        "age": 56
    }
}