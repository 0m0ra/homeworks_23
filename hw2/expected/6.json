{"status": "JSONDecodeError", "message": "Input file is empty"}